femtologging/pure.py

```
@@ -1,6 +1,19 @@
 from __future__ import annotations
 
 
+class FemtoLogger:
+    """Simplistic Python implementation used when the Rust module is missing."""
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def log(self, level: str, message: str) -> str:
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)